zcash_client_backend/CHANGELOG.md

```diff
@@ -33,6 +35,7 @@ and this library adheres to Rust's notion of
 
 ### Changed
 - MSRV is now 1.77.0.
+- Migrated to `zcash_protocol 0.4.1`, `zcash_primitives 0.20.0`, `zcash_keys 0.5.0`.
```

Comment:
I don't think we need to mention point release migrations, because all point releases are compatible in the sense that someone on `0.4.0` can trivially use `0.4.1` instead, but couldn't use `0.5`. This is also why I generally only include the first two digits of the version in "migrated to" messages (as those are the backwards-incompatible and/or MSRV-changing digits).

Reply:
In this case, we actually do require 0.4.1; that's the only reason I called it out.

Reply:
We do, but the fact that we do is not a breaking change to downstream users of our crate.